### DISCUSSION TITLE
The scripts just hanging out without error warning

### DISCUSSION THREAD

**Discussion by rifkycc** [matiasdelellis/facerecognition#713] [Q&A]
Hello, I've been attempting to run the facerecognition program on my server by running the command 'sudo -u www-data php /var/www/thearchiver.cloud/occ face:background_job -u rifky -t 400'. 

![image](https://github.com/matiasdelellis/facerecognition/assets/30744244/cd49428c-5d88-4215-9241-435ca454f60f)

Despite the program's status indicating '11 image left in queue, estimated done in few seconds,' but it remains unresponsive without giving any error, even after 8 hours. How can I troubleshoot this issue? I already look in FAQ, but didn't find anything that explained my problem. Also, I can't found any related error in nextcloud logs

![image](https://github.com/matiasdelellis/facerecognition/assets/30744244/8737f791-9147-4c7c-a482-045d87aa4a45)

**Comment by matiasdelellis** [comment]
Hi @rifkycc 
These images are probably from another account.

Mm.. See face:stats comand.
